### DISCUSSION TITLE
KinematicCollision should be renamed to CharacterCollision

### DISCUSSION THREAD

**Discussion by MeBadDev** [godotengine/godot-proposals#6047] [Engine Core]
Now `KinematicBody` is renamed to `CharacterBody`, we should probably rename `KinematicCollision` to `CharacterCollision` to avoid confuses and it will probably make things easier, does that make any sense?

**Comment by outobugi** [comment]
Afaik the move_and_collide, which returns that object, is now a method of PhysicsBody, so it makes sense to have 'Kinematic' in the name.
